docs/metrics/create-metric-tags.md

```diff
@@ -1,11 +1,9 @@
 ---
 title: Creating Metric Tags
 sidebar_label: Creating Metric Tags
-slug: /metrics/create-metric-tags
+slug: /metrics/metrics-201/create-metric-tags
 ---
```

Comment:
I thought we were going to leave the urls alone so links didnt break, but just rename them inline and in the sidebar

I'm okay with going this route, long term its not a concern and doing the change sooner is better

But while we are at it:
/metrics/metrics- is a bit redundant, how about just /metrics/201/create-metric-tags?